src/systems/system_projection.C

```diff
@@ -1888,7 +1888,7 @@ void BoundaryProjectSolution::operator()(const ConstElemRange & range) const
                             if (!dof_is_fixed[j])
                               freej++;
                           }
-                        Fe(freei) += phi[i][qp] * fineval * JxW[qp];
+                        Fe(freei) += (fineval * phi[i][qp]) * JxW[qp];
```

Comment:
why this change?

Reply:
Why is it in that commit?  Because I stupidly had it sitting in my working tree when I did a `git commit -a` on the matrix stuff.  Can't believe I didn't notice it in the list of changed files while writing the commit message.

I'll take it out of the PR entirely; it didn't come to anything in the end.

Why did I make the change in the first place?  Because I was in the process of preparing projection code (perhaps the most often cut-and-paste-and-tweaked code in all of libMesh and MOOSE...) for refactoring, and my first step in the workflows here was generally "make any trivial changes as necessary to get the [Copy/Paste Detector](https://pmd.github.io/latest/pmd_userdocs_cpd.html) to detect longer duplicate blocks", rather than relying on my own reading to be sure that two blocks are duplicates the whole way through.

Reply:
Thanks for the review, BTW.  Repeatedly looking at duplicate-except-for-whitespace-and-comment code blocks makes my eyes want to glaze over, so while I tried to be careful about bugs I'm definitely happier with more eyes on the diffs.